electron-app/cypress.env.json

```diff
@@ -0,0 +1,9 @@
+{
+  "TZ": "UTC",
+  "ROTKEHLCHEN_ENVIRONMENT": "test",
+  "PASSWORD": 1234,
+  "ETH_ADDRESS": "0x52bc44d5378309EE2abF1539BF71dE1b7d7bE3b5",
+  "BTC_ADDRESS": "3PFo18vaPMSXFTt6zUDGk3UoPjD56QLXjh",
+  "BITTREX_API_KEY": "539939bffed348299c6a859ca3f9a41e",
```

Comment:
you can also use gemini here. They have an actual sandox with data and the API key I got in the tests has many trades and has balances:

https://github.com/rotki/rotki/blob/c43e1931e0151c44d67533a4a87e1b2b43c86a3a/rotkehlchen/tests/fixtures/exchanges/gemini.py#L5-L6